edwin/client/static/js/components/Timeline.js

```diff
@@ -82,6 +99,22 @@ Timeline.propTypes = {
   params: React.PropTypes.object.isRequired,
 };
 
+class Indicator extends React.Component {
+  render() {
+    let state = this.props.state.toJS().state;
```

Comment:
`this.props.state` is kind of a funny thing to say, this React components generally have a `this.state` which stores other kinds of data. I'm not sure what other name to use for it though.